src/IssueViz.Security/IssueVizSecurityPackage.cs

```diff
@@ -41,7 +42,9 @@ namespace SonarLint.VisualStudio.IssueVisualization.Security
     [Guid("D7D54E08-45E1-49A6-AA53-AF1CFAA6EBDC")]
     [ProvideMenuResource("Menus.ctmenu", 1)]
     [ProvideToolWindow(typeof(HotspotsToolWindow), MultiInstances = false, Transient = true, Style = VsDockStyle.Tabbed, Window = VsWindowKindErrorList, Width = 700, Height = 250)]
-    [ProvideToolWindow(typeof(TaintToolWindow), MultiInstances = false, Transient = true, Style = VsDockStyle.Tabbed, Window = VsWindowKindErrorList, Width = 700, Height = 250)]
+    [ProvideToolWindow(typeof(TaintToolWindow), MultiInstances = false, Transient = false, // Note: Transient must false when using ProvideToolWindowVisibility
```

Comment:
`Transient = false` is the default, maybe it's not worth setting it at all and avoid the comment this way.

Reply:
I think it's clearer with the comment as it's not obvious and not state in the VS docs.

Reply:
Ok (it is in the docs though https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.shell.providetoolwindowattribute.transient?view=visualstudiosdk-2019#Microsoft_VisualStudio_Shell_ProvideToolWindowAttribute_Transient)

Reply:
It's not in the docs that the value has to be false, otherwise it seems to mess up the operation of the `ProvideToolWindowVisibilityAttribute`.